client/src/game/ui/drawUI.ts

```diff
@@ -1077,7 +1077,7 @@ function drawScoreArea() {
     });
     terminateButton.tooltipName = "kill";
     terminateButton.tooltipContent =
-      'Vote to terminate the game (click again to cancel the vote).<br><span style="padding-left: 30px;">Double click to terminate immediately.</span>';
+      'Vote to terminate the game (click again to cancel the vote).<br /><span style="padding-left: 30px;">Double click to terminate immediately.</span>';
```

Comment:
The `<br />` is a relic of xhtml and has been long abandonned.

https://www.google.com/search?q=%3Cbr%3E+vs+%3Cbr+%2F%3E&oq=%3Cbr%3E+vs+%3Cbr+%2F%3E

The span's role is to indent the second paragraph:

![image](https://user-images.githubusercontent.com/2622043/145351224-d6479b48-6aaf-4cab-909d-f269b6d55356.png)

vs

![image](https://user-images.githubusercontent.com/2622043/145351402-70aff24b-d041-482d-8e78-26afae97082a.png)

I found it unecessary to to place the style in hanabi.css because it's a dynamically created tooltip.